source/services/encryption/EncryptionService.stories.tsx

```diff
@@ -4,7 +4,9 @@ import styled from "styled-components/native";
 import StoryWrapper from "../../components/molecules/StoryWrapper";
```

Comment:
Do we have a reason for not writing jest unit tests for this?

Reply:
Not really, but we currently lack the expertise. I can give it a shot though, but it might take some time.

Reply:
I have rewritten the stories tests into Jests tests now 🙂.